src/components/table/table.visualspec.js

```diff
@@ -0,0 +1,12 @@
+import { percySnapshot } from '@percy/puppeteer';
+
+describe('Table', () => {
+  beforeAll(async () => {
+    await page.goto(`${HOST}/table`);
```

Comment:
Nitpick: we can just return the promise. No need for async/await. But it is kind of a 🤷‍♀️